xmppserver/src/test/java/org/jivesoftware/Fixtures.java

```diff
@@ -89,13 +98,15 @@ public static XMPPServer mockXMPPServer() {
             .when(xmppServer).createJID(any(String.class), nullable(String.class), any(Boolean.class));
         doReturn(mockXMPPServerInfo()).when(xmppServer).getServerInfo();
         doReturn(mockIQRouter()).when(xmppServer).getIQRouter();
+        doReturn(mockConnectionManager()).when(xmppServer).getConnectionManager();
 
         return xmppServer;
     }
 
     public static XMPPServerInfo mockXMPPServerInfo() {
         final XMPPServerInfo xmppServerInfo = mock(XMPPServerInfo.class, withSettings().lenient());
         doReturn(XMPP_DOMAIN).when(xmppServerInfo).getXMPPDomain();
+        doReturn(XMPP_DOMAIN).when(xmppServerInfo).getHostname();
```

Comment:
Is it worth creating a new constant with a different value? I can see issues where the hostname and XMPP domain, not necessarily the same, could get confused with each other.

Reply:
Good idea. Done.